code/datums/shuttles.dm

```diff
@@ -109,62 +109,77 @@
 		M.movement_force = movement_force.Copy()
 
 /datum/map_template/shuttle/emergency
+	prefix = "_maps/shuttles/emergency/"
 	port_id = "emergency"
 	name = "Base Shuttle Template (Emergency)"
```

Comment:
It would probably be better to have the file name build from
_maps/shuttles/[shuttle_type]/[filename].dmm
and then have the dreammaker type paths mirror the file structure